spec/models/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Anny', post_counter: 1) }
+
+  it 'Name must not be blank' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'Name must not be blank' do
+    subject.name = 'Anny'
+    expect(subject).to be_valid
+  end
+
+  it 'Post counter must be an integer' do
```

Comment:
Some of the `tests are failing with the error message below:

![image](https://user-images.githubusercontent.com/10332499/167263213-3b75241a-0296-42a3-bfce-2b62cda93a19.png)

Reply:
@micheaol Please these are not the tests I wrote for this milestone.

Kindly review the tests for this milestone using the command `rspec spec/models` to review the current milestone.

Thanks for understanding.

Reply:
I have updated the README accordingly.

Reply:
Hi @brytebee,

I am only highlighting one of the errors received after running your tests.